projects/yolox-pose/configs/yolox-pose_s-poseota_8xb32-300e_coco.py

```diff
@@ -0,0 +1,11 @@
+_base_ = './yolox-pose_s_8xb32-300e_coco.py'
```

Comment:
I suggest only inheriting base configs like dataset, but not another complete config file. Multi-level inheritance sometimes leads to bad user experiences.